documentation/docs/guides/creating-plans.md

```diff
@@ -0,0 +1,305 @@
+The Goose desktop doesn't have a `plan` keyword. If you want the desktop to create a plan for you, you need to use a prompt like:
+
+```
+"Hey Goose, can you create a plan to convert my CLI project into a locally hosted web page that gives me input fields for each CLI command I can run? Please don't start the actual work"
+```
+Unless you ask Goose to "create a plan", it might just start into the project work. 
+:::
+
+The Goose CLI's plan mode differs from desktop planning in a key way: it's interactive, asking clarifying questions to understand your project before creating a plan. In contrast, the desktop version creates plans immediately based on your description alone, making assumptions that might miss critical project details.
+
+## Set your planner provider and model
+In some workflows, it can be helpful to use one LLM for planning and a different one for execution. For example, GPT-4.1 tends to excel at strategic planning and breaking down complex tasks into clear, logical steps. On the other hand, Claude Sonnet 3.5 is particularly strong at writing clean, efficient code and following instructions precisely. By using GPT-4.1 to plan and Claude to execute, you can play to the strengths of both models and get better results overall.
+
+The Goose CLI plan mode uses two configuration values:
+
+- `GOOSE_PLANNER_PROVIDER`: Which provider to use for planning
+- `GOOSE_PLANNER_MODEL`: Which model to use for planning
+
+For example, you might configure:
+```bash
+export GOOSE_PLANNER_PROVIDER=<my-chosen-provider>
+export GOOSE_PLANNER_MODEL=<my-chosen-model>
+```
+If these aren't set, Goose will use your default provider and model settings. You might want to set different planning models if you find certain models are better at breaking down tasks into clear steps. However, your default model configuration is usually sufficient.
+
+To verify that the planner provider is set, input the following terminal command:
+
+```bash
+~ goose info -v
+```
+
+In this example, the `info` command returns the current configuration and the path to the configuration file.  
+
+```bash
+Goose Version:
+  Version:          1.0.18
+
+Goose Locations:
+  Config file:      /Users/alincoln/.config/goose/config.yaml
+  Sessions dir:     /Users/alincoln/.local/share/goose/sessions
+  Logs dir:         /Users/alincoln/.local/state/goose/logs
+
+Goose Configuration:
+  GOOSE_PROVIDER: anthropic
+  GOOSE_MODEL: claude-3.5-sonnet
+  GOOSE_PLANNER_PROVIDER: openai
+  GOOSE_MODE: smart_approve
+  GOOSE_PLANNER_MODEL: gpt-4.1
+```
+
+ If either `GOOSE_PLANNER_PROVIDER` or `GOOSE_PLANNER_MODEL` are not set, `GOOSE_PROVIDER` and `GOOSE_MODEL`are used to build your plan.  
+
+## Describe your project
+While Goose can handle complex project descriptions, it works best with clear, concise ones. Focus on stating your project's purpose and desired outcomes. If these aren't clear, Goose will ask clarifying questions until it fully understands your goals. 
```

Comment:
could the examples start really simple first like here's a simple plan and then do one with a more complex plan where it will ask clarifying questions